docs/plan-for-production.md

```diff
@@ -0,0 +1,332 @@
+
+
+When to consider a commercial solution such as UCP
+
+
+
+
+# Planning for Swarm in production
+

```

Comment:
@moxiegirl All of the HA examples refer to AWS availability zones as do the diagrams. I'm happy to leave it that way (it's easier). But there was a convo on slack yesterday about referencing AWS too much might look like we're taking sides. Any thoughts?